libnvme/nvme.i

```diff
@@ -335,6 +336,7 @@ struct nvme_ctrl {
   char *traddr;
   char *trsvcid;
   %extend {
+    char *dhchap_host_key:
     char *dhchap_key;
```

Comment:
Wouldn't make sense to rename `dhcap_key` to `dhcp_ctrl_key` as in the C code part as well?

Reply:
Compat stuff. When we're renaming it in the python binding all existing python programs will need to be updated. With this they continue working.

Reply:
Okay, so it's another todo on the 2.x list.